src/test/java/org/akhq/repositories/AccessControlRepositoryTest.java

```diff
@@ -44,6 +44,19 @@ void findAllByUser() throws ExecutionException, InterruptedException {
         );
     }
 
+    @Test
+    void findAllBySecondUser() throws ExecutionException, InterruptedException {
+        var searchResult = aclRepository.findByPrincipal(KafkaTestCluster.CLUSTER_ID, AccessControl.encodePrincipal("test:toto"), Optional.empty());
+        assertEquals("test:toto", searchResult.getPrincipal());
+        assertEquals(2, searchResult.getAcls().size());
+        assertEquals(2, searchResult
```

Comment:
I am not sure whether this test case is necessary or not.

Reply:
@LittleWat - I added to make sure few more principal patterns other than one starting with user* still works after my changes.

Reply:
Thanks! I just thought both the result(`searchResult.getAcls().size()` and `searchResult.stream().filter(acl -> acl.getOperation().getPermissionType() == AclPermissionType.ALLOW).count())` ) are `2`, so it might be okay to remove the latter one. This is just a thought so that you can ignore it. 🙇